person.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'corrector'
+require_relative 'rental'
+
+class Person
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @corrector = Corrector.new
+    @name = validate_name(name)
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def of_age?
+    age >= 18
```

Comment:
- [ ] your instance variable which you created on line 12 is `@age` not `age` which you have used please kindly correct this piece of code :+1: